player/command.c

```diff
@@ -6052,6 +6052,11 @@ static void cmd_run(void *p)
         .detach = true,
     };
     struct mp_subprocess_result res;
+    MP_VERBOSE(mpctx, "Starting subprocess: [%s", args[0]);
+    char **arg = &args[1];
+    while(*arg)
+        MP_VERBOSE(mpctx, ", %s", *arg++);
+    MP_VERBOSE(mpctx, "]\n");
```

Comment:
wouldn't it make more sense to add this to `mp_subprocess2` via an mp_log argument?

Reply:
I didn't want to change signature of `mp_subprocess2` and it is more changes, because it is per platform.

Reply:
done